app-accessibility/yasr/yasr-0.6.9-r1.ebuild

```diff
@@ -39,11 +51,6 @@ src_configure() {
 src_install() {
 	emake DESTDIR="${D}" install || die
```

Comment:
(emake doesn't need die anymore in EAPI6.)
https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-130001r2